exchanges/binance/binance_wrapper.go

```diff
@@ -110,9 +112,30 @@ func (b *Binance) SetDefaults() {
 			},
 			WithdrawPermissions: exchange.AutoWithdrawCrypto |
 				exchange.NoFiatWithdrawals,
+			KlineCapabilities: kline.ExchangeCapabilities{
+				SupportsDateRange: true,
+				SupportsIntervals: true,
+			},
 		},
 		Enabled: exchange.FeaturesEnabled{
 			AutoPairUpdates: true,
+			KlineCapabilities: kline.ExchangeCapabilities{
+				Intervals: map[string]bool{
+					"onemin":     true,
```

Comment:
lack of presence indicates unsupported interval by the exchange

And yeah i did start at one point having it as a const across exchanges
But I was unhappy with the naming since i can't recall it OneMin

I tried a few including jstu having it as kline.OneMin.String() & kline.OneMinStr
( https://github.com/xtda/gocryptotrader/commit/3e64b8fec04620ed1d6f925ddd79ecf175f11415#diff-bcea53bbc99926eaaebe9a6d1db1bb6f )

I might just end up going with the .String() version